README.md

```diff
@@ -0,0 +1,133 @@
+# pyramid-googleauth
```

Comment:
I'm gonna send a PR to rewrite the `README` based on a test app I'm developing locally. Will send it later